src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/AddExpenseCommandParserTest.java

```diff
@@ -59,9 +63,16 @@ public void parse_allFieldsPresent_success() {
     @Test
     public void parse_optionalFieldsMissing_success() {
         // zero categories
-        Expense expectedExpense = new TransactionBuilder(BUBBLE_TEA_2).withCategories().buildExpense();
+        Expense expectedExpenseWithNoCategories = new TransactionBuilder(BUBBLE_TEA_2).withCategories().buildExpense();
```

Comment:
```suggestion
        Expense expectedExpenseWithNoCategories = new TransactionBuilder(BUBBLE_TEA_2).buildExpense();
```

Redundant method call.